io.github.cudatext.CudaText-Qt5.yml

```diff
@@ -0,0 +1,107 @@
+      - type: archive
+        url: https://github.com/Alexey-T/Emmet-Pascal/archive/refs/tags/2020.09.05.tar.gz
+        sha256: 'cba0d68fee260a5f7aff51933014fe53ea3eb71f6d8349ff660b8b2a04028eaf'
+        dest: src/Emmet-Pascal
+      - type: archive
+        url: https://github.com/Alexey-T/CudaText/archive/refs/tags/1.139.5.tar.gz
+        sha256: '0c9929bf5870347399d017e104153de0a9a5bfd7428a8631e72bcf4481e2ea48'
+        dest: src/CudaText
+    buildsystem: simple
+    build-commands:
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/bgrabitmap/bgrabitmap/bgrabitmappack.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/EncConv/encconv/encconv_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/ATBinHex-Lazarus/atbinhex/atbinhex_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/ATFlatControls/atflatcontrols/atflatcontrols_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/ATSynEdit/atsynedit/atsynedit_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/ATSynEdit_Cmp/atsynedit_cmp/atsynedit_cmp_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/EControl/econtrol/econtrol_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/ATSynEdit_Ex/atsynedit_ex/atsynedit_ex_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/Python-for-Lazarus/python4lazarus/python4lazarus_package.lpk"
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/Emmet-Pascal/emmet/emmet_package.lpk"
+      # Replace references to /usr/share/cudatext with correct path, prior to compiling CudaText
+      - sed -i -e"s/\/usr\/share\/cudatext/\\$FLATPAK_DEST\/share\/cudatext/g" src/CudaText/app/proc_globdata.pas
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/CudaText/app/cudatext.lpi"
+      - install -Dm755 src/CudaText/app/cudatext -t $FLATPAK_DEST/bin
+      # Install all data and plugins
+      - mkdir -p $FLATPAK_DEST/share/cudatext
+      - cp -r src/CudaText/app/data $FLATPAK_DEST/share/cudatext/
+      - cp -r src/CudaText/app/py $FLATPAK_DEST/share/cudatext/
+      - cp -r src/CudaText/app/settings_default $FLATPAK_DEST/share/cudatext/
+      # Desktop file and icon - desktop file needs to have the icon name corrected first (and carriage-returns removed)
+      - sed -i -e"s/\r//g" src/CudaText/setup/debfiles/cudatext.desktop
+      - sed -i -e"s/Icon=cudatext-512/Icon=io.github.cudatext.CudaText\-Qt5/" src/CudaText/setup/debfiles/cudatext.desktop
+      - install -D src/CudaText/setup/debfiles/cudatext.desktop $FLATPAK_DEST/share/applications/io.github.cudatext.CudaText-Qt5.desktop
+      - install -D src/CudaText/setup/debfiles/cudatext-512.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/io.github.cudatext.CudaText-Qt5.png
```

Comment:
the appdata file isn't installed, which why the build failed on x86_64.

Reply:
Where do I install the appdata file to?

Reply:
Weird that didn't stop it building the flatpak on my local machine

Reply:
flatpak-builder doesn't check. Flathub buildbot does.

Reply:
Never mind I found where it should go, fixing...

Reply:
> Where do I install the appdata file to?

in `$FLATPAK_DEST/share/metainfo`

Reply:
Fixed.